portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/searchtuning/SearchExperiences.testcase

```diff
@@ -0,0 +1,601 @@
+@component-name = "portal-search"
+definition {
+
+	property portal.release = "true";
+	property portal.suite.search.engine = "elasticsearch7";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Search Experiences";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			BlogsEntry.tearDownCP();
+
+			JSONUser.tearDownNonAdminUsers();
+
+			WebContent.tearDownCP();
+
+			SearchExperiences.tearDownCP();
+
+			Page.tearDownCP();
+		}
+	}
+
+	test AssertAddCopyAndDeleteSearchBlueprint {
+		property test.name.skip.portal.instance = "SearchExperiences#AssertAddCopyAndDeleteSearchBlueprint";
```

Comment:
I just noticed that Search Experiences is scoped per Portal virtual instance. Is there a reason we're still setting `test.name.skip.portal.instance` in our tests?

Reply:
We were getting some failures in our suite when running the tests with virtual instances. Skipping instances was supposed to be a quick fix as we got our suite running regularly against the dev branch.

Reply:
ah okay, do you know if we can remove that property now? I can try running some tests with it removed.

Reply:
During our meeting, we ended up discussing this and decided we should remove this property and address any issues that arise from having our tests run with instances